frontend/app/components/account-form.js

```diff
@@ -0,0 +1,90 @@
+import { action } from "@ember/object";
+import AccountValidations from "../validations/account";
+import lookupValidator from "ember-changeset-validations";
+import Changeset from "ember-changeset";
+import { inject as service } from "@ember/service";
+import { tracked } from "@glimmer/tracking";
+import BaseFormComponent from "./base-form-component";
+import { bind } from "@ember/runloop";
+
+export default class AccountForm extends BaseFormComponent {
+  @service store;
+  @service router;
+
+  @tracked selectedTeam;
+  @tracked allTeams = this.store.findAll("team");
```

Comment:
assignableTeams ?
a term we introcuded in SWEB :)
it's never all teams, is it? just the ones that the user has access to